assets/javascripts/discourse/initializers/init.js.es6

```diff
@@ -1,8 +1,155 @@
+        return headerLinksConfig.split("|").map(headerItem => {
+          const headerConfig = headerItem.split(",").map(value => value.trim());
+          const [label, title, href, className, target, keep] = headerConfig;
+
+          return {
+            label,
+            title,
+            href,
+            className,
+            target: target === "self" ? "" : "_blank",
+            keep: keep !== "keep" ? "" : "keep"
+          };
+        });
+      };
+
+      const debtcollectiveLinks = headerLinks(
+        Discourse.SiteSettings.debtcollective_header_links
+      );
+
+      // Add links to the header
+      debtcollectiveLinks.forEach(link => {
+        api.decorateWidget("header-buttons:before", helper => {
+          return helper.h(
+            `li.header-link.${link.className}.${link.keep}`,
+            helper.h(
+              "a",
+              {
+                href: link.href,
+                title: link.title,
+                target: link.target
+              },
+              link.label
+            )
+          );
+        });
+      });
+
+      api.reopenWidget("hamburger-menu", {
+        panelContents() {
+          const { currentUser } = this;
+          const results = [];
+
+          // Insert debtcollective links at the top of the menu
+          if (debtcollectiveLinks.length > 0) {
+            results.push(
+              this.attach("menu-links", {
+                name: "debtcollective-links",
+                heading: true,
+                contents: () => {
+                  return debtcollectiveLinks.map(l => {
+                    return this.attach("link", {
+                      href: l.href,
+                      className: l.className,
+                      rawLabel: l.label
+                    });
+                  });
+                }
+              })
+            );
+          }
+
+          let faqUrl = this.siteSettings.faq_url;
+          if (!faqUrl || faqUrl.length === 0) {
+            faqUrl = Discourse.getURL("/faq");
+          }
+
+          const prioritizeFaq =
+            this.settings.showFAQ &&
+            this.currentUser &&
+            !this.currentUser.read_faq;
+
+          if (prioritizeFaq) {
+            results.push(
+              this.attach("menu-links", {
+                name: "faq-link",
+                heading: true,
+                contents: () => {
+                  return this.attach("priority-faq-link", { href: faqUrl });
+                }
+              })
+            );
+          }
+
+          if (currentUser && currentUser.staff) {
+            results.push(
+              this.attach("menu-links", {
+                name: "admin-links",
+                contents: () => {
+                  const extraLinks = flatten(
+                    applyDecorators(this, "admin-links", this.attrs, this.state)
+                  );
+                  return this.adminLinks().concat(extraLinks);
```

Comment:
Would we want to add the tools admin link here too if the user is in the right groups?

Reply:
This is [the same original method](https://github.com/discourse/discourse/blob/3584af4e51d0874e5a725c1bedd3118940c1896e/app/assets/javascripts/discourse/widgets/hamburger-menu.js.es6#L258-L317). We are extending this method to include our links at the top

Reply:
Sure. Do we have a ticket for that?

Reply:
No, actually let's ignore this, we already have an Admin link for Discourse and don't want to add another Admin link here because it may be confusing. Is more intuitive if they go to the tools and then to the admin section.